### ISSUE TITLE
Why need the function of 'blob_utils.get_loss_gradients'?

### ISSUE THREAD

**Issue by yw155** [facebookresearch/Detectron#785]
Hi @ir413, I would like to ask you that why there is a function of 'blob_utils.get_loss_gradients'. From my understanding, after defining the loss, we just need to deploy the op of 'AddGradientOperators' as follows:

loss = model.AveragedLoss(xent, "loss")
model.AddGradientOperators([loss])

So why do we need the function of 'blob_utils.get_loss_gradients' in the Detectron?

Thank you.
